wgpu/tests/common/mod.rs

```diff
@@ -187,18 +187,28 @@ pub fn initialize_test(parameters: TestParameters, test_function: impl FnOnce(Te
 
     let missing_features = parameters.required_features - adapter_features;
     if !missing_features.is_empty() {
+        #[cfg(all(target_arch = "wasm32", feature = "webgl"))]
+        delete_html_canvas();
+
+        // TODO: we probably should use log crate here for logging also to wasm console
         println!("TEST SKIPPED: MISSING FEATURES {:?}", missing_features);
```

Comment:
@cwfitzgerald How do you feel about adding and using [console_log](https://docs.rs/console_log/latest/console_log/) as a dev-dependency here? 
It looks like it's already used in the examples.

Reply:
That's fine